src/test/java/com/amihaiemil/docker/RtImageTestCase.java

```diff
@@ -274,7 +291,42 @@ public void tagErrorIfResponseIs500() throws Exception {
             new AssertRequest(
                 new Response(HttpStatus.SC_INTERNAL_SERVER_ERROR)
             ),
-            URI.create("https://localhost")
+            URI.create("https://localhost"),
+            DOCKER
         ).tag("myrepo/myimage", "mytag");
     }
+
+    /**
+     * RtImage can run itself.
+     * @throws Exception If something goes wrong.
+     */
+    @Test
+    public void runsItselfOk() throws Exception {
+        final AtomicBoolean started = new AtomicBoolean(false);
+        final Container container = Mockito.mock(Container.class);
+        Mockito.doAnswer(
+            invocation -> {
+                started.set(true);
+                return null;
+            }
+        ).when(container).start();
+        final Containers containers = Mockito.mock(Containers.class);
+        Mockito.doReturn(container)
+            .when(containers)
+            .create(Mockito.eq("image123"));
+        Mockito.doReturn(containers).when(DOCKER).containers();
+        MatcherAssert.assertThat(
+            new RtImage(
+                Mockito.mock(JsonObject.class),
+                Mockito.mock(HttpClient.class),
+                URI.create("http://localhost/images/image123"),
+                DOCKER
+            ).run(),
+            Matchers.notNullValue()
```

Comment:
@llorllale instead of ``notNullValue()``, can you compare the result of ``run()`` with the mock ``container`` above? It should be the exact same reference, right?

Reply:
@amihaiemil done